src/main/java/org/sagebionetworks/web/client/widget/entity/tabs/AbstractTablesTab.java

```diff
@@ -380,7 +380,7 @@ private void configureVersionAlert() {
 					goToLatestSnapshot,
 					isLinkToCurrentSnapshotEnabled,
 					linkToCurrentSnapshotTooltipText);
-		} else if (!version.equals(latestSnapshotVersionNumber)) {
+		} else if (version != null && !version.equals(latestSnapshotVersionNumber)) {
```

Comment:
I don't think this can happen in the real world, but it was causing some error messages in tests.